org.scala-ide.sdt.core/src/org/scalaide/ui/ScalaImages.scala

```diff
@@ -37,4 +37,8 @@ object ScalaImages {
   val DESC_SETTINGS_OBJ: ImageDescriptor = OSGiUtils.getImageDescriptorFromCoreBundle("/icons/full/obj16/settings.gif")
 
   val ADD_METHOD_PROPOSAL: Image = JavaPluginImages.DESC_MISC_PUBLIC.createImage()
+  val PRIVATE_VAR :ImageDescriptor = JavaPluginImages.DESC_FIELD_PRIVATE
```

Comment:
Formatting is broken (colon, no space before, space after). Please do format action on your newly introduced code.